production/ksonnet/loki/boltdb_shipper.libsonnet

```diff
@@ -0,0 +1,143 @@
+{
+  local pvc = $.core.v1.persistentVolumeClaim,
+  local volumeMount = $.core.v1.volumeMount,
+  local container = $.core.v1.container,
+  local statefulSet = $.apps.v1.statefulSet,
+  local service = $.core.v1.service,
+  local containerPort = $.core.v1.containerPort,
+
+  _config+:: {
+    // configurable pvc sizes
+    ingester_pvc_size: '5Gi',
+    querier_pvc_size: '10Gi',
+    compactor_pvc_size: '10Gi',
+
+    boltdb_shipper_shared_store: error 'must define boltdb_shipper_shared_store',
+    index_period_hours: 24,
+    loki+: {
+      chunk_store_config+: {
+        write_dedupe_cache_config:: {},
```

Comment:
This is set by another libsonnet so in order to disable it only on boltdb when you do `loki + boltdb_shipper` this is required in jsonnet

Reply:
Or do you mean we actually should not care because it will automatically disabled ? https://grafana.com/docs/loki/latest/operations/storage/boltdb-shipper/#write-deduplication-disabled

Reply:
We do not use that config so it was better to remove it instead of using the default which would have some values set. It avoids setting a wrong expectation who doesn't know much about it.